src/test/runTest.ts

```diff
@@ -7,7 +7,11 @@ import {
   downloadAndUnzipVSCode,
 } from "vscode-test";
 
-const extensionDependencies = ["pokey.parse-tree", "ms-toolsai.jupyter"];
+const extensionDependencies = [
+  "pokey.parse-tree",
+  "ms-toolsai.jupyter",
+  "scalameta.metals",
```

Comment:
In case you're curious, this line was what fixed CI.  The problem is that without this extension, VSCode doesn't know about the `scala` language id, so when our test runner creates a language with that language id, it just ignores us and treats it as `plaintext`, hence the error message where cursorless tried to process scope types for `plaintext`